### ISSUE TITLE
getFileList($page) not working

### ISSUE THREAD

**Issue by iseva** [uploadcare/uploadcare-php#103]
The paginator is not working.

```
$page = 2;
$files = $api->getFileList($page);
```

An integer can not be passed as an array is expected. I

`public function getFileList($options = array()) { … }
`
Is this an oversight as the documentation states otherwise? Has the latest code been pushed?

**Comment by dmitry-mukhin** [comment]
what lib version are you using?

**Comment by dmitry-mukhin** [comment]
Yes, readme is outdated. Now `getFileList` returns `FileIterator` and you shouldn't use explicit pagination, just iterate through those files:

```php
$files = $api->getFileList(/* some filter options */);
foreach ($files as $file) {
  // do something with each file. pagination is done implicitly for you.
  echo $file->getUrl();
}

**Comment by dmitry-mukhin** [comment]
fixed in 0c0acd9
